molecule/install-remove-waf/converge.yml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
We could maybe merge both WAF/DoS uninstall tests into the same Molecule scenario?